frontend/src/MuiForms/TagSelectorBeta.tsx

```diff
@@ -0,0 +1,78 @@
+import { Autocomplete, Box, Chip, Stack, TextField, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+
+export default function TagSelector(props: any) {
+  const { onChange, value: currentValue, label, formContext } = props
+
+  const _onChange = (_event: React.SyntheticEvent<Element, Event>, newValues: any) => {
+    onChange(newValues.map((value) => value))
+  }
+
+  const theme = useTheme()
+
+  return (
+    <>
+      {formContext.editMode && (
+        <>
+          <Typography sx={{ fontWeight: 'bold' }}>{label}</Typography>
+          <Autocomplete
+            multiple
+            isOptionEqualToValue={(option: any, value: any) => option === value}
+            value={currentValue || ''}
+            onChange={_onChange}
+            options={[]}
+            freeSolo
+            renderTags={(value: readonly string[], getTagProps) =>
+              value.map((option: string, index: number) => (
+                // eslint-disable-next-line react/jsx-key
+                <Chip variant='outlined' label={option} {...getTagProps({ index })} />
+              ))
+            }
+            renderInput={(params) => (
+              <TextField
+                {...params}
+                size='small'
+                sx={{
+                  input: {
+                    color: theme.palette.mode === 'light' ? 'black' : 'white',
+                  },
+                  label: {
+                    WebkitTextFillColor: theme.palette.mode === 'light' ? 'black' : 'white',
```

Comment:
This applies to all uses of `WebkitTextFillColor` in this file. See: https://github.com/gchq/Bailo/pull/691/files#r1318463150